src/gridstack-engine.ts

```diff
@@ -670,6 +670,10 @@ export class GridStackEngine {
 
       if (collide) {
         needToMove = !this._fixCollisions(node, nn, collide, o); // check if already moved...
+      } else if (activeDrag && !collide) {
```

Comment:
this isn't the right fix - for one thing !collide isn't necessary since if() above. Also when dealing with subgrid creation (code above) this will mess things up (collide = undefined line 667)

I like what you are trying to do, but needs some tweaking...